src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+
+import {
+  APPOINTMENT_STATUS,
+  UNABLE_TO_REACH_VETERAN_DETCODE,
+} from '../../utils/constants';
+import { sentenceCase } from '../../utils/formatters';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+
+import { getPatientTelecom } from '../../services/appointment';
+import { selectExpressCareRequestById } from '../redux/selectors';
+import PageLayout from './AppointmentsPage/PageLayout';
+
+function ExpressCareDetailsPage({ appointment }) {
+  const appointmentDate = moment.parseZone(appointment?.start);
+  const canceled = appointment?.status === APPOINTMENT_STATUS.cancelled;
+
+  useEffect(() => {
+    scrollAndFocus();
+    document.title = `Express Care request on ${appointmentDate.format(
+      'dddd, MMMM D, YYYY',
+    )}`;
+  }, []);
+
+  if (!appointment) {
+    return <Redirect to="/" />;
+  }
+
+  const unableToReachVeteran =
+    appointment.cancelationReason?.text === UNABLE_TO_REACH_VETERAN_DETCODE;
+
+  return (
+    <PageLayout>
```

Comment:
I put the PageLayout component in here because I need to make sure the Route renders this component in a way that passes through the route props, which I need in the `mapStateToProps` function below.